toolbox/library/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ endif()
 # ===================
 
 # YARP blocks
+list(APPEND WBT_BLOCKS "YarpRpc")
```

Comment:
RPC is an acronym, perhaps it could make sense to always spell it as YarpRPC? On the other hand also YARP is an acronym, so it could make sense to leave as it is.

Reply:
As discussed with @diegoferigo we can leave it as `YarpRpc`